app/views/customer_service/index.html.erb

```diff
@@ -270,12 +270,14 @@
                     <% if contact.claims_post %>
                       <tr>
                         <th scope="row">Address:</th>
-                        <td><%= simple_format(contact.claims_post) %></td>
+                        <td><%= simple_format(contact.claims_post, {}, wrapper_tag: 'span') %></td>
```

Comment:
`simple_format` will add paragraph tags (a block-level element) around text and `wrapper_tag` is wrapping _those_ in a `<span>` (an inline element) which creates a functional-but-kind-of-weird markup structure:

```html
<span>
  <p>…</p>
</span>
```

Reply:
I checked the markup and it doesn't do that! Maybe an older version did?

Reply:
![screen shot 2017-12-06 at 9 33 51 am](https://user-images.githubusercontent.com/29409348/33666764-fad586a4-da68-11e7-8536-b6d91b6a3ad4.png)

Reply:
Oh, derp. That's on me. I misinterpreted the documentation. _Carry on!_